static/js/tab_bar.js

```diff
@@ -99,13 +99,13 @@ function make_tab_data() {
             tabs.push(make_tab("Search results", false));
         }
 
-        // Third breadcrumb item for stream-subject naarrows
+        // Third breadcrumb item for stream-topic narrows
```

Comment:
Is this stream-topic/subject something special?

Reply:
It is just the regular stream & topic that is refered to. So I updated the comment since the code here now do not use the name subject anymore.

Reply:
👍